Some small improvements to MFSA YAML styling and function

pmac opened the pull request:
A real effort to make these things look nice is coming in a near-future sprint.

pmac commented:
Unfortunately I also corrected the indentation in `security.less`. Please forgive the diff :(

@alexgibson r?

alexgibson commented on bedrock/security/templates/security/partials/cve.html:
```diff
@@ -1,19 +1,21 @@
-    {%- endfor %}
-  </ul>
-{% endif %}
+<div class="cve">
```
Nit: could make this a bit more semantic by using a `<section>` element instead of a `<div>`.

alexgibson commented on bedrock/security/templates/security/partials/cve.html:
```diff
@@ -1,19 +1,21 @@
+  <h5>Description</h5>
+  {{ description|markdown }}
+  {% if bugs %}
+    <h5>References</h5>
```
Nit: This could probably be an `<h6>`.

pmac replied:
It's not actually a sub heading of "Description", it's just indented because of the `if` tag. In that case `h5` is correct right?

alexgibson replied:
Ah you're quite right, I misread 👍

alexgibson commented on media/css/security/security.less:
```diff
@@ -57,6 +57,26 @@
+                padding: 6px 6px 6px 120px;
+                .level {
+                    font-size: inherit;
+                    line-height: 12px;
```
`line-height` should be unit-less, a value of `1` here should work.

alexgibson commented:
At small viewports some of the long headings break out of the container. Adding `word-wrap: break-word;` to the headings should fix things (not ideal, but a quick fix).

![wrod-wrap](https://cloud.githubusercontent.com/assets/400117/19569438/a7c1a430-96ed-11e6-9642-3d9c6b219f1b.png)

pmac commented:
Updated things from your suggestions. The `section` element inherited some unfortunate styles, so I had to do a bit more tweaking to make those go away, but I think it looks pretty good.

I also removed the border above the section titles and increased the top margin.

alexgibson commented on media/css/security/security.less:
```diff
@@ -57,6 +57,27 @@
         letter-spacing: normal;
     }
+    .cve {
+        margin: 2.5em 0 0 2.5em;
```
Should this be `1em` on the left, to keep the content in line? `2.5em` feels a bit odd, especially at smaller screen sizes.
